### ISSUE TITLE
[Keyboard navigation - MakeCode editor] Visually not clear when Blocks-Javascript-Python options highlighted

### ISSUE THREAD

**Issue by microbit-lucy** [microsoft/pxt-microbit#5538]
**Describe the bug**
If you navigate the MakeCode editor using a keyboard, pressing the tab key steps through the menu options. However, you cannot see when the Blocks, Javascript & Python options are highlighted (i.e. the :focus-visible styling isn't clear). It fine for other menu items: Home, Share, Help, Settings, Account as a white border around the menu. 

- Blocks option: becomes subtly greyer but very hard to see
- Javascript option: is greyed out already, does get slightly darker but very hard to see
- Python option: can't be seen at all as it's in a sub-menu that doesn't expand when that option is hightlight, only the arrow to this menu is subtly highlight but that is hard to see. 

**To Reproduce**
1. Go to 'https://makecode.microbit.org/#editor'
2. Press tab several times 

https://github.com/microsoft/pxt-microbit/assets/102724242/ae8008f5-f289-4dc5-b1dc-2f3b88d92570

**Expected behavior**
Ensure the :focus-visible styling is clear for each of the 3 coding language options so the user can see which is selected.

**Comment by microbit-robert** [comment]
The video belows demonstrates one way to improve the focus-visible styles for the editor toggle button. This removes the subtle background changes that have transition (more like hover effects) as these don't seem to make sense for keyboard navigation. Feedback is very welcome.

https://github.com/user-attachments/assets/d3b9b087-5a2c-4011-a10b-785678300e6b

**Comment by kmcnaught** [comment]
The focus highlights seem reasonable. 

The remaining animation seems a bit slow/clunky, the way it highlights the Javascript option first and then has a small delay even before the slide animation. How much of that is an artefact of the screen recording?

**Comment by abchatra** [comment]
This is fixed.
